src/frontend/AppContainerWrapper.tsx

```diff
@@ -100,6 +102,11 @@ const AppContainerWrapper = () => {
     const previousRouteName = getRouteName(previousState);
     const currentRouteName = getRouteName(currentState);
 
+    //sets practice bar on of off depedning if route name is included in NO_PRACTICE_BAR array
+    setHidePracticeBar(
+      currentRouteName !== null && NO_PRACTICE_BAR.includes(currentRouteName)
```

Comment:
optional but could extract this into a helper function in this file since you do an identical thing in the initializer for `hidePracticeBar`. Something like:

```ts
const shouldHidePracticeBar = (route: string | null) => route !== null && NO_PRACTICE_BAR.includes(route)
```